doc/api/cli.md

```diff
@@ -448,6 +448,18 @@ changes:
 
 Specify the maximum size, in bytes, of HTTP headers. Defaults to 16KB.
 
+### `--max-worker-threads=n`
+<!-- YAML
+added: REPLACEME
+-->
+
+Specify the maximum number of worker threads that should be created for
+this Node.js process. Defaults to one less than four times the number of
+available CPUs.
```

Comment:
One less? If that’s intended to account for the main thread, I wouldn’t do it. The workload on the main thread is usually sufficiently different in nature to spawn as many Workers as CPUs for a given thread pool, not CPUs - 1.

Reply:
Yep, I was going back and forth on this also as to whether accounting for the main thread would be needed. Will adjust that